discovery/docker_discovery.go

```diff
@@ -43,6 +44,7 @@ func NewDockerDiscovery(endpoint string, svcNamer ServiceNamer, ip string) *Dock
 		containerCache: NewContainerCache(),
 		serviceNamer:   svcNamer,
 		advertiseIp:    ip,
+		sleepInterval:  SLEEP_INTERVAL,
```

Comment:
Sory I didn't notice before, but it occurs to me that this should become `DefaultSleepInterval` since it's just the default now. Also an opportunity to fix the bad all caps constant.